src/clr-addons/components.clr-addons.scss

```diff
@@ -52,6 +60,20 @@ body,
     .dropdown-menu {
       z-index: map-get($clr-layers, sidepanel) - 2;
     }
+
+    @media screen and (max-width: map-get($clr-grid-breakpoints, sm)) {
+      margin-bottom: 0.5rem;
+    }
+  }
+
+  @media screen and (max-width: map-get($clr-grid-breakpoints, sm)) {
+    display: block;
+    overflow: hidden;
+    padding: 0 0.5rem;
+
+    > * {
+      float: left;
```

Comment:
why is this needed? if you do this, no item can be right aligned in the header e.g. command bar (although there wouldn't be much alignment on such small screens, but we force everyone to left alignment)
just remove the "display: block" and "overflow:hidden" from content-header and the h2 should already reduce it's size so no overflow should happen on the content-header right?

Reply:
Buttons and command bars are not intended to be right aligned for screens widths < 576px. They need to be wrapped in a second line.

Reply:
Just talked with Dieter Schrottshammer and it's not intended to wrap the header at all